x/oracle/abci.go

```diff
@@ -70,26 +71,52 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
 
 		//---------------------------
 		// Do miss counting & slashing
-		voteTargetsLen := len(voteTargets)
+		denomMap := map[string]map[string]bool{}
```

Comment:
this can be `map[string]map[string]struct{}{}` since we never use the value or set anything but `true`. set the value to `struct{}{}`, slightly more efficient